stock_picking_validate_jobcan_wf/models/stock_picking.py

```diff
@@ -25,9 +25,22 @@ def _is_outgoing(self):
             "transit",
         ] and self.location_dest_id.usage not in ["internal", "transit"]
 
+    def _receipt_return_picking(self):
+        self.ensure_one()
+        if not self.move_ids:
+            return False
+        move = self.move_ids[0]
+        if move.origin_returned_move_id and move._is_out():
+            return True
+        return False
```

Comment:
I think the code can be leaner. Technically speaking, Jobcan WF should be skipped if ALL items in the outgoing transfer are returns to the vendor.

Reply:
This adjustment is needed for not showing and requiring of `jobcan_wf_number` for receipts return.
https://github.com/qrtl/axls-custom/blob/3720a67608535eea4fdf2b787340386c8f26f041/stock_picking_validate_jobcan_wf/views/stock_picking_views.xml#L15-L18